README.adoc

```diff
@@ -0,0 +1,32 @@
+= Spring Cloud Task
+
+Is a project centered around the idea of processing on demand.  A user is able to develop
+a “task” that can be deployed, executed and removed on demand, yet the result of the
+process persists beyond the life of the task for future reporting.
+
+
+== Requirements:
+
+* Java 7 or Above
+
+== Build:
+
+[source,shell,indent=2]
+----
+$ mvn clean install
```

Comment:
Great question.  In short the task annotation will be applied to Spring Boot Applications (CommandLineRunner/ApplicationRunners).  Thus we need to have the users of the annotation pull this this projects artifacts into their project as a dependency in a POM file.